docs/cli/index.md

```diff
@@ -48,7 +48,7 @@ Commands for managing accounts
 **Synopsis**
 
 Commands for managing accounts. An account is a pair of a private key and a public key.
-Starport uses accounts to interact with the Starport Network blockchain, use an IBC relayer, and more.
+Ignite CLI uses accounts to interact with the Ignite CLI Network blockchain, use an IBC relayer, and more.
```

Comment:
@ilgooz I wasn't sure if we needed updates in this file?? Is this content elsewhere in the codebase?